packages/instrumentation-langchain/src/instrumentation.ts

```
@@ -16,226 +16,90 @@
-  private unpatchRunnablesModule(moduleExports: typeof RunnablesModule) {
-    this._diag.debug(`Unpatching @langchain/core/runnables`);
-
-    this._unwrap(moduleExports.Runnable.prototype, "invoke");
-    return moduleExports;
+  private patchCallbackManager(CallbackManager: unknown) {
+    const callbackManagerAny = CallbackManager as Record<string, unknown>;
+    
+    if (callbackManagerAny._configureSync && !callbackManagerAny._traceloopPatched) {
+      const originalConfigureSync = callbackManagerAny._configureSync;
+      
+      const self = this;
```

ESLint

[error] Unexpected aliasing of 'this' to local variable.

(@typescript-eslint/no-this-alias)